bin/wmagent-resource-control

```diff
@@ -95,32 +95,19 @@ def createOptionParser():
 
     return myOptParser
 
-def getCMSSiteInfo(pattern):
+def getCMSSiteInfo(pattern=''):
     """
     _getCMSSiteInfo_
 
     Query SiteDB for the site and SE names matching the pattern.  Return a
     dictionary keyed by site name.
     """
-    nameSEMapping = {}
     print "Querying SiteDB for site information..."
 
-    siteDB = SiteDBJSON()
-    for seName in siteDB.cmsNametoList(pattern, "SE", "result.json"):
-        cmsNames = siteDB.seToCMSName(seName)
-        for cmsName in cmsNames:
-            if cmsName in nameSEMapping:
-                nameSEMapping[cmsName].append(seName)
-            else:
-                nameSEMapping[cmsName] = [seName]
-
+    pattern = pattern.replace('*', '.*').replace('%', '.*')
     if options.addDisk:
-        nameSEMapping = dict((k, list(set(v))) for k, v in nameSEMapping.iteritems()
-                        if not k.endswith("_Disk"))
-    else:
-        nameSEMapping = dict((k, list(set(v))) for k, v in nameSEMapping.iteritems())
-
-    return nameSEMapping
+        return SiteDBJSON().PSNtoPNNMap('^(?!.*_Disk\s*$)' + pattern)
```

Comment:
Shouldn't this be for opposite condition options.addDisk == False?
I think we always set the addDisk options True now so it might not be needed to check this condition at all.